introduction.md

```diff
@@ -8,17 +8,17 @@ In the following, we go through a story that describes an application delivery l
 2. The _application operator_ deploys instances of that application, and configures it with operational traits, such as autoscaling;
 3. The _infrastructure operator_ decides which underlying technology is used to handle the deployment and operations.
 
-## Application developer: Write and test code
+## Application developer: write and test code
 
-The story begins with the application developers who create an application, such as an online shopping application. They know how to write and test the code. The application program takes a few parameters such as log level and HTTP port. To let the developers focus on implementing the application's business logic, we have application operators (either human or automated operation platforms) take care of operational tasks. This provides a "serverless" experience to application developers: they only need to develop and package the application, and then deliver it to application operators to operate
+The story begins with the application developers who create an application, such as an online shopping application. They know how to write and test the code. The application program takes a few parameters such as log level and HTTP port. To let the developers focus on implementing the application's business logic, we have application operators (either human or automated operation platforms) take care of operational tasks. This provides a "serverless" experience to application developers: they only need to develop and package the application, and then deliver it to application operators to operate.
```

Comment:
what is changed here?

Reply:
A stop character at the end of the sentence.